packages/quantic/package.json

```diff
@@ -98,6 +98,8 @@
     "force-app/main/default/cspTrustedSites/*",
     "force-app/main/default/staticresources/*",
     "scripts/npm",
-    "docs/out/quantic-docs.json"
+    "docs/out/quantic-docs.json",
```

Comment:
if `scripts` are also in the files here, can we not just remove the `.npmignore` file?

Reply:
No because here in files we are specifying `scripts/npm`, that's the only script folder we need `npm`.

under the folder `scripts` there are other sub folders that we wanna ignore, they are ignored using the ` .npmignore` file

Reply:
oh you're right I forgot we had more than the npm script. All good then!